backend/layers/processing/process_validate_atac.py

```diff
@@ -0,0 +1,224 @@
+import hashlib
+
+from backend.common.utils.corpora_constants import CorporaConstants
+from backend.layers.business.business_interface import BusinessLogicInterface
+from backend.layers.common.entities import (
+    ARTIFACT_TO_EXTENSION,
+    CollectionVersionId,
+    DatasetArtifactId,
+    DatasetArtifactType,
+    DatasetConversionStatus,
+    DatasetStatusKey,
+    DatasetValidationStatus,
+    DatasetVersionId,
+)
+from backend.layers.common.ingestion_manifest import IngestionManifest
+from backend.layers.processing.exceptions import ConversionFailed, ValidationAtacFailed
+from backend.layers.processing.process_logic import ProcessingLogic
+from backend.layers.thirdparty.s3_provider_interface import S3ProviderInterface
+from backend.layers.thirdparty.schema_validator_provider import SchemaValidatorProviderInterface
+from backend.layers.thirdparty.uri_provider import UriProviderInterface
+
+
+class ProcessValidateATAC(ProcessingLogic):
+    def __init__(
+        self,
+        business_logic: BusinessLogicInterface,
+        uri_provider: UriProviderInterface,
+        s3_provider: S3ProviderInterface,
+        schema_validator: SchemaValidatorProviderInterface,
+    ) -> None:
+        super().__init__()
+        self.business_logic = business_logic
+        self.uri_provider = uri_provider
+        self.s3_provider = s3_provider
+        self.schema_validator = schema_validator
+
+    def create_atac_artifact(
+        self,
+        file_name: str,
+        artifact_type: DatasetArtifactType,
+        dataset_version_id: DatasetVersionId,
+        processing_status_key: DatasetStatusKey,
```

Comment:
why is this an arg if there's only one ATAC_STATUS?

Reply:
i'll remove and inline it.